wg-advisors/why/why_branding_licencing.md

```diff
@@ -1,13 +1,68 @@
 # Why we should look after the project's brand and licencing
 
-TODO:
-
 ## Summary
 
+The Apache Software Foundation's most important asset is a combination of branding and license that is
+used by all Apache projects. The license is the Apache License, Version 2.0, and the branding is based
+on the project name. The licence is a permissive open source license that allows the software to be used
+by anyone for any purpose, and the branding is a set of rules that ensure that the software is recognized
+as an Apache project and that the project is recognized as part of the Apache Software Foundation.
+
+The software that ASF projects produce is released under the Apache License, Version 2.0, which is a permissive
+open source license that allows the software to be used by anyone for any purpose and it is important that
+we do not limit anyone from using the software in any way. And while we allow anyone to use the software,
+and redistribute it, we do not allow anyone to claim that they are the "owners" of the software, or that
+they are controlling the project.
+
+This is why it is important to ensure that the project's brand is not used in a way that could cause confusion
+or mislead users on who is owning or controlling the project. This includes ensuring that the project's name
+is not used in a way that could imply that another party is controlling the project - assuring
+vendor neutrality.
```

Comment:
Perhaps a link here to one of the docs where we discuss vendor neutrality in greater detail.

Reply:
Good point. Added links to [Project Independence](https://community.apache.org/projectIndependence.html)